src/main/java/sirius/kernel/xml/StructuredNode.java

```diff
@@ -48,6 +50,16 @@
 
     private Node node;
 
+    /**
+     * Handles any namespace logic within the xml.
+     * <p>
+     * We need to add a namespace context to query nodes which belong to a namespace.
+     * <p>
+     * E.g. <a:tag></a:tag> would belong to the namespace a and we need to add it to query it.
+     */
+    @Part
+    private static NamespaceContext namespaceContext;
```

Comment:
why? this logic is only used here - new ConfigBasedNamespaceContext() should be all good.